frontend/src/queries/schema.ts

```diff
@@ -2090,3 +2083,28 @@ export interface ActorsPropertyTaxonomyQuery extends DataNode<ActorsPropertyTaxo
 export type ActorsPropertyTaxonomyQueryResponse = AnalyticsQueryResponseBase<ActorsPropertyTaxonomyResponse>
 
 export type CachedActorsPropertyTaxonomyQueryResponse = CachedQueryResponse<ActorsPropertyTaxonomyQueryResponse>
+
+export enum AssistantMessageType {
+    Human = 'human',
+    Assistant = 'ai',
+    Visualization = 'ai/viz',
+}
+
+export interface HumanMessage {
+    type: AssistantMessageType.Human
+    content: string
+}
+
+export interface AssistantMessage {
+    type: AssistantMessageType.Assistant
+    content: string
+}
+
+export interface VisualizationMessage {
+    type: AssistantMessageType.Visualization
+    plan?: string
+    reasoning_steps?: string[] | null
+    answer?: ExperimentalAITrendsQuery
+}
+
+export type RootAssistantMessage = VisualizationMessage | AssistantMessage | HumanMessage
```

Comment:
If I rename this type to AssistantMessageRoot, the Pydantic generator doesn't use the AssistantMessageType enum🙃 It just replaces the values with the type name, so the type field of AssistantMessage becomes `Literal['AssistantMessage']`.